exporters/jaeger/thrift-gen/Collector.cpp

```diff
@@ -0,0 +1,481 @@
+/**
+ * Autogenerated by Thrift Compiler (0.14.0)
```

Comment:
I am not fully aware of how `.thrift` files should be handled in general for the build. For `.proto` we compile them as part of our build process. Should same be done for thrift files, and then not check-in cpp and headers ?

Reply:
There are examples for both, check in `.thrift` file (OpenTelemetry-Python) and check in the generated files (OpenTelemetry-DotNet and [Jaeger-client-cpp](https://github.com/jaegertracing/jaeger-client-cpp/tree/master/src/jaegertracing/thrift-gen)). As the Jaeger [`.thrift` files](https://github.com/jaegertracing/jaeger-idl/) are stable and not in active development (this is very different for OTLP), so I'd prefer check in the generated files to avoid the cpp codegen for each build.